Makefile

```diff
@@ -132,79 +116,114 @@ fleetctl: .prefix .pre-build .pre-fleetctl
-# GO_TEST_EXTRA_FLAGS: Used to specify other arguments to `go test`.
 .debug-go-tests:
 ifeq ($(PKG_TO_TEST), "")
-		@echo "Please specify one or more packages to debug with argument PKG_TO_TEST=\"/path/to/pkg/1 /path/to/pkg/2\"..."; 
+		@echo "Please specify one or more packages to debug. See '$(TOOL_CMD) help run-go-tests' for more info."; 
 else
 		@echo Debugging tests with command:
 		dlv test ${dlv_test_pkg_to_test} --api-version=2 --listen=127.0.0.1:61179 ${DEBUG_TEST_EXTRA_FLAGS} -- -test.v -test.run=${TESTS_TO_RUN} ${GO_TEST_EXTRA_FLAGS} 
 endif
 
-# Command to run specific tests in development.  Can run all tests for one or more packages, or specific tests within packages.
+.help-short--run-go-tests:
+	@echo "Run Go tests in specific packages"
+.help-long--run-go-tests:
+	@echo Command to run specific tests in development. Can run all tests for one or more packages, or specific tests within packages.
+.help-options--run-go-tests:
+	@echo "PKG_TO_TEST=\"pkg1 pkg2...\""
+	@echo "Go packages to test, e.g. \"server/datastore/mysql\". Separate multiple packages with spaces."
+	@echo "TESTS_TO_RUN=\"test\""
+	@echo Name specific tests to debug in the specified packages. Leave blank to debug all tests in the specified packages.
+	@echo "GO_TEST_EXTRA_FLAGS=\"--flag1 --flag2...\""
+	@echo "Arguments to send to \"go test\"."
 run-go-tests:
 	@MYSQL_TEST=1 REDIS_TEST=1 MINIO_STORAGE_TEST=1 SAML_IDP_TEST=1 NETWORK_TEST=1 make .run-go-tests GO_TEST_MAKE_FLAGS="-v"
 
+.help-short--debug-go-tests:
+	@echo "Debug Go tests in specific packages (with Delve)"
+.help-long--debug-go-tests:
+	@echo Command to run specific tests in the Go debugger. Can run all tests for one or more packages, or specific tests within packages.
+.help-options--debug-go-tests:
+	@echo "PKG_TO_TEST=\"pkg1 pkg2...\""
+	@echo "Go packages to test, e.g. \"server/datastore/mysql\". Separate multiple packages with spaces."
+	@echo "TESTS_TO_RUN=\"test\""
+	@echo Name specific tests to debug in the specified packages. Leave blank to debug all tests in the specified packages.
+	@echo "GO_TEST_EXTRA_FLAGS=\"--flag1 --flag2...\""
+	@echo "Arguments to send to \"go test\"."
 debug-go-tests:
 	@MYSQL_TEST=1 REDIS_TEST=1 MINIO_STORAGE_TEST=1 SAML_IDP_TEST=1 NETWORK_TEST=1 make .debug-go-tests 
 
 # Command used in CI to run all tests.
+.help-short--test-go:
+	@echo "Run the Go tests (all packages and tests -- used in CI)"
 test-go: dump-test-schema generate-mock 
 	make .run-go-tests PKG_TO_TEST="./cmd/... ./ee/... ./orbit/pkg/... ./orbit/cmd/orbit ./pkg/... ./server/... ./tools/..."
 
 analyze-go:
 	go test -tags full,fts5,netgo -race -cover ./...
 
+.help-short--test-js:
+	@echo "Run the JavaScript tests"
 test-js:
 	yarn test
 
+.help-short--test:
+	@echo "Run the full test suite (lint, Go and Javascript -- used in CI)"
 test: lint test-go test-js
 
+.help-short--generate:
+	@echo "Generate and bundle required all code"
```

Comment:
Given that we're moving this around, probably better to revise this description to be more accurate, as we have a bunch of generate commands and this only builds bindata on the go side (vs. commands that update mocks, schema, etc.)